uber/models.py

```diff
@@ -1827,7 +1833,11 @@ def getter(self, params=None, *, bools=(), checkgroups=(), allowed=(), restricte
             if id == 'None':
                 inst = model()
             else:
-                inst = self.query(model).filter_by(id=id).one()
+                try:
+                    inst = self.query(model).filter_by(id=id).one()
+                except NoResultFound:
+                    self.rollback()
+                    raise HTTPRedirect('../preregistration/error_attendee_not_found?id={}', id)
```

Comment:
If I understand correctly, `getter()` is used to access ANY model - not just attendees. This should be a more generic page with a more generic message, if possible.